gsd-web/src/boot/env.js

```diff
@@ -0,0 +1,22 @@
+/*
+  Extracts any envvars that are prefixed with the GSD_ namespace
+  so they can be overlayed a top the dotenv base configuration in
+  <project root> .env
+
+  This file is loaded during the quasar.config.js which is not transpiled
+  by babel so this module is define to be require nodejs style.
+
+  Normally this would extend quasar boot but since this targets a browser
+  dotenv requires node fs, os, and path which are unavailable.
```

Comment:
This has me worried that the env interpolation is being baked in at compile time, and that runtime might not see this properly if the compile time was missing things but runtime includes them. I think I see now why you were testing this with the build and start commands.

Reply:
@ninjapanzer 

> // Embed client ID to allow login requests <- I was never able to verify this comment.
What I mean is that when I set a envvar for this and then ran build. I didn't see the value in the dist source. So it seems that even though this env property is in the build declaration something isn't right. Or I am misunderstanding how this is supposed to work. Cause I think this always just adds the ID to the process.env.

Okay, yeah I think my fears are true then. This does some weird edge cases.

Reply:
Later after writing the comment I committed and digging through the documentation I learned that the quasar.config.js that the cli loads happens at the node level for the express purposes of allowing access to `fs` and things like that. Once this file is loaded and it boots into Vue its all file.

But what the `env` property does is pass things into process.env so I think we are doing double work really. If the env is set by dotenv then this is already done. No "cache" like thing is produced but this envvar and value are used during the build process. For the code to run it also needs to be in the shell env and passed into the process.env like normal.

This section of the config appears to be a shortcut for allowing you to mix multiple source of config and make sure they are all handled in a consistent manner. Be it a file loaded, hardcoded strings, or transposition from the current env. We could probably just omit this whole thing as I am not convinced it is used as intended.